### ISSUE TITLE
Schedule activities for getting the dynamic reactive motion

### ISSUE THREAD

**Issue by iron76** [kzts/athletebot#4]
Here is a list of steps that should be completed in order to implement the jumping/reactive motion on the Athletebot using its YARP interface and the inverse-dynamics controller (@kzts and @nskw).
- [x] Include the FT sensors yarp-plugin in the URDF model so that we can read FT sensors from a YARP port. This is related to solving #3. Even if the real robot does not have all the FT sensors (2 on feet and 2 on legs) we should include all of them in the URDF.
- [x] Configure the yarp-wholebodyinterface (https://github.com/robotology/yarp-wholebodyinterface) using the same URDF file used for simulating the Athletebot in Gazebo. 
- [x] Test on Gazebo the inverse-dynamics controllers. Two solution are possible at this stage. Either we use the `(C++) torqueBalancing` (https://github.com/robotology/codyco-modules/tree/master/src/modules/torqueBalancing) or the `(Simulink) torqueBalancing` (https://github.com/robotology-playground/WBI-Toolbox-controllers/tree/master/controllers/torqueBalancing).
- [ ] Start playing with the configuration trajectories of the `torqueBalancing` controller to perform a dynamic-reactive motion. It is foreseen that the parameters to play with are the center of mass acceleration and the reference posture configuration. This activity can really on this module (https://github.com/robotology/codyco-modules/tree/master/src/modules/torqueBalancingReferencesGenerator) which should be documented enough to be easy to use.

**Comment by kzts** [comment]
@iron76 Are there any models including `yarp-wholebodyinterface` plugin?
If there are, I want to see them as references for configuring `yarp-wholebodyinterface`.

**Comment by iron76** [comment]
Yes sure. Please have a look at https://github.com/robotology/yarp-wholebodyinterface/tree/master/examples/yarpWholeBodyInterfaceExample. For compiling it as usual create a `build` directory, then `cmake ..`  and finally `make`.

**Comment by kzts** [comment]
@iron76 Thank you for advice. I could make `yarpWholeBodyInterfaceExample`.
But I don't know how to use it. Please tell me which page I should read.

**Comment by francesco-romano** [comment]
Hi @kzts.
[Here](https://github.com/lornat75/Teaching/tree/master/code/wbi) there is an example  (implementing a PD + gravity compensation) using the yarpWholeBodyInterface.
While in the code there is some comments it does not explain in full details the "yarp part" (i.e. how to create a Module, how to use the ResourceFinder).

**Comment by kzts** [comment]
@francesco-romano Thank you for your suggestions. I'll try it.

**Comment by kzts** [comment]
I made the sdf file inculding ft sensors, got messages like following and I didn't get any error messages 

```
Sensor default::athletebot_fixed::ATHLETE_ROBOT_CY::l_foot_fixed::left_foot_ft was happily found!
```

But our robot model unfortunately exploded on gazebo. 
I want Nori-sensei to check this tomorrow.

**Comment by iron76** [comment]
@kzts Can you push the new model into the git repository so that I can give it a try?

**Comment by kzts** [comment]
@iron76 I pushed it to new branch https://github.com/kzts/athletebot/tree/model_ft
